x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/PyTorchModelIT.java

```diff
@@ -111,30 +118,144 @@ public void unsetLogging() throws IOException {
         RAW_MODEL_SIZE = Base64.getDecoder().decode(BASE_64_ENCODED_MODEL).length;
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/pull/75759")
+    @After
+    @SuppressWarnings("unchecked")
+    public void cleanup() throws IOException {
```

Comment:
This should already be handled by the reset API integration. I don't think this is necessary.

Reply:
I added MLRestTestCleaner to do the cleanup